recipes/easybuild-framework/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "easybuild-framework" %}
+{% set version = "3.1.0" %}
+{% set md5 = "f2d06240467f06c98fb2e9243ce5d8ef" %}
+
+package:
+  name: {{ name }} 
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/fd/dd/83d8ef6a981c9e953f523328d5fba23b1be84fb7640a2c1d8b8e66b07102/{{name}}-{{version}}.tar.gz
+  md5: f2d06240467f06c98fb2e9243ce5d8ef
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools >=0.6
+    - pyyaml
+    - coloredlogs
+    - humanfriendly
+    - vsc-install >=0.9.19
+    - vsc-base >=2.5.4
+
+  run:
+    - python
+    - setuptools >=0.6
+    - pyyaml
+    - coloredlogs
+    - humanfriendly
+    - vsc-install >=0.9.19
+    - vsc-base >=2.5.4
+
+test:
+  imports:
+    - easybuild
+    - easybuild.framework
+    - easybuild.framework.easyconfig
+    - easybuild.framework.easyconfig.format
+    - easybuild.toolchains
+    - easybuild.toolchains.compiler
+    - easybuild.toolchains.fft
+    - easybuild.toolchains.linalg
+    - easybuild.toolchains.mpi
+    - easybuild.tools
+    - easybuild.tools.deprecated
+    - easybuild.tools.job
+    - easybuild.tools.module_naming_scheme
+    - easybuild.tools.package
+    - easybuild.tools.package.package_naming_scheme
+    - easybuild.tools.repository
+    - easybuild.tools.toolchain
+
+about:
+  home: http://hpcugent.github.com/easybuild
+  license: GPLv2
+  summary: 'The EasyBuild framework supports the creation of custom easyblocks that implement support for installing particular (groups of) software packages.'
+  license_family: GPL2
```

Comment:
Is there a license file in the source that we can add to the package using `license_file`? Maybe the answer is no.

xref: https://github.com/hpcugent/easybuild/issues/301